packages/cli/config-management/docs-configuration/src/DocsConfiguration.ts

```diff
@@ -9,13 +9,37 @@ export interface DocsConfiguration {
     favicon: ImageReference | undefined;
     colors: FernRegistry.docs.v1.write.ColorsConfig;
     navbarLinks: FernRegistry.docs.v1.write.NavbarLink[] | undefined;
+    typography?: TypographyConfig;
 }
 
 export interface Logo {
     reference: ImageReference;
     href: FernRegistry.docs.v1.write.Url | undefined;
 }
 
+export interface FontConfig {
+    name: string;
+    path: string;
```

Comment:
since we don't use this path, I could see us deleting it

Reply:
We don’t use it? How does the user supply a font

Reply:
I believe we do use it to resolve the actually font source file. @dsinghvi

Reply:
Can we change it from string to AbsoluteFilePath? This is something we introduced because there were a bunch of bugs with us mixing up relative and absolute filepaths

Reply:
there is one on line 23 (right below this line for `AbsoluteFilePath`) which is why i was saying we don't/shouldn't use it

Reply:
Ahhh lol

Reply:
@codebender828 basically at `loadWorkspace` time we should convert the `path` that the user gave us into an `AbsoluteFilePath` that can later be used by `publishDocs.ts`

Reply:
After thinking about this, I came up with a solution: I have decided to make the `name` property optional, and the `path` will be required. If no `name` is not supplied, we shall default it to use a randomly generated `name` that will be used to reference the font in the eventually injected CSS. My options for consideration would be the `absoluteFilePath`, or a `randomId` like the `FileId`.

It would be useful in case, the developer wants to inspect what the actual name of the font being used in the published documentation is. I also think it will set us up in the future for better customization in the future as the font name will be used as a token value. So I suggest we keep the `name` option. But it does not need to be explicitly supplied for the docs definition to work. So we keep the definition thin, but provide a bit more visibility if they want to provide a font name. (Will be more useful later on during the local dev setup.)

Reply:
that sounds good -- can you add this in the docs for the fern definition `docs-config` ?

Reply:
Done!

Reply:
We can delete `path` right? We just need `absolutePath`